libraries/debug-certificate-manager/src/CertificateManager.ts

```diff
@@ -70,20 +75,35 @@ interface ISubjectAltNameExtension {
   altNames: readonly IAltName[];
 }
 
-interface IAltName {
+/**
+ * Fields for a Subject Alternative Name of type DNS Name
+ */
+interface IDnsAltName {
   type: 2;
   value: string;
 }
+/**
+ * Fields for a Subject Alternative Name of type IP Address
+ */
+interface IIPAddressAltName {
+  type: 7;
+  ip: string;
+}
+type IAltName = IDnsAltName | IIPAddressAltName;
 
 /**
  * Options to use if needing to generate a new certificate
  * @public
  */
 export interface ICertificateGenerationOptions {
   /**
-   * The DNS Subject names to issue the certificate for.
+   * The DNS Subject names to issue the certificate for. Defaults to ['localhost'].
    */
   subjectAltNames?: ReadonlyArray<string>;
+  /**
+   * The IP Address Subject names to issue the certificate for. Defaults to ['127.0.0.1'].
+   */
+  subjectIpAddresses?: ReadonlyArray<string>;
```

Comment:
```suggestion
  subjectIPAddresses?: ReadonlyArray<string>;
```

Though obviously that's just personal preference